### PR TITLE
Features/menu buttons

### PR DESCRIPTION
**Pull request by pjumruspun**
### Upload button
- Functionality hasn't been added, but can choose file for now. As the only way to currently play the file is to move the file into `public` folder, which is not the intended way to do (the intended way is to use backend or cloud storage in later sprints).
- This can be implemented by calling a function from **child** `FileButton.js` to **parent** `Bar.js` to **grandparent** `App.js`

### Change volume
- There is an unknown bug causing the video to not change volume if the video is playing

### CODE REVIEW

File: front-end/src/components/FileButton.js [hunk 1]
```diff
@@ -29,9 +47,17 @@ var ButtonStyle = {
         keepMounted
         open={Boolean(anchorEl)}
         onClose={handleClose}
-	anchorPosition={{ vertical: "top", horizontal: "left" }}
+        anchorPosition={{ vertical: "top", horizontal: "left" }}
       >
-        <MenuItem onClick={handleClose}>Import</MenuItem>
+        <MenuItem for="file-upload">
+          <label for="avatar">Import </label>
```
Comment (warintorntantirittisak) [inline front-end/src/components/FileButton.js]: in order to hide the Choose File button and only show Import.

`<label className="custom-file-upload">
          <input type="file" id="video-input" hidden
            accept="video/mp4, video/webm"
            onChange={handleImport} />
          <i className="fa fa-cloud-upload" /> Import
        </label>`

**Review by warintorntantirittisak — APPROVED** [review]

**Comment by warintorntantirittisak** [comment]
just found out that the page crashes if import->select a file->import again->cancel when in a file browser

File: front-end/src/components/FileButton.js (hunk 1 above)
Comment (pjumruspun) [reply front-end/src/components/FileButton.js]: I used your exact code to hide the choose file button now, thank you.

**Comment by pjumruspun** [comment]
> just found out that the page crashes if import->select a file->import again->cancel when in a file browser

Canceling causes `file` to be `undefined`, and it crashes because it tries to `console.log(file.name)` so I just check if the file is not undefined first. It's now fixed

**Pull request merged**